telethon/client/chats.py

```diff
@@ -1167,6 +1184,7 @@ async def get_permissions(
             for participant in chat.participants.participants:
                 if participant.user_id == user.id:
                     return custom.ParticipantPermissions(participant.participant, True)
+            return None
```

Comment:
If the user is not a participant I think this should trigger an error. Otherwise forgetting to check if there's a result value will cause `NoneType has no attribute…` errors which are more confusing. For example, `GetParticipant` will raise `errors.UserNotParticipantError`, so I think this should do the same.

```suggestion
            raise errors.UserNotParticipantError(None)
```

Reply:
but there no `UserNotParticipantError` in errors folder. should i create new error in `common.py`?

Reply:
There is one, it's generated automatically from the list of [`errors.csv`](https://github.com/LonamiWebs/Telethon/blob/master/telethon_generator/data/errors.csv).